needles/fbe_keyboard.json

```
@@ -13,6 +13,14 @@
       "height": 604,
       "type": "match"
     },
+    {
+      "": "Layouts other that English (US) may change",
```

```suggestion
      "": "Layouts other than English (US) may change",
```

Suggestion: Also add a FIXME to a Phab task for improving the list of keyboard layouts, so this exclude can potentially eventually be removed.

---

I don't think we should have a FIXME here: the layouts changing should not cause the whole openQA suite to fail.

The linked ticket, https://phabricator.endlessm.com/T25604, now includes an AC for adding a test that explicitly checks the layouts are sensible.